GameServer/User.cpp

```diff
@@ -2922,50 +2944,23 @@ void CUser::AppendNoticeEntry(Packet & pkt, uint8 & elementCount, const char * m
 void CUser::AppendExtraNoticeData(Packet & pkt, uint8 & elementCount)
 {
 	string message;
-			if (g_pMain->m_byExpEventAmount < 1000)
-	{
-		g_pMain->GetServerResource(IDS_BAR5, &message, g_pMain->m_byExpEventAmount);
-		AppendNoticeEntry(pkt, elementCount, message.c_str(), "Knight Online 2.0"); 
-	}
-			if (g_pMain->m_byExpEventAmount < 1000)
-	{
-		g_pMain->GetServerResource(IDS_BAR1, &message, g_pMain->m_byExpEventAmount);
-		AppendNoticeEntry(pkt, elementCount, message.c_str(), "Fire Drake"); 
-	}
-			if (g_pMain->m_byExpEventAmount < 1000)
-	{
-		g_pMain->GetServerResource(IDS_BAR2, &message, g_pMain->m_byExpEventAmount);
-		AppendNoticeEntry(pkt, elementCount, message.c_str(), "Bifrost"); 
-	}
-			if (g_pMain->m_byExpEventAmount < 1000)
-	{
-		g_pMain->GetServerResource(IDS_BAR3, &message, g_pMain->m_byExpEventAmount);
-		AppendNoticeEntry(pkt, elementCount, message.c_str(), "Border Defance War"); 
-	}
-			if (g_pMain->m_byExpEventAmount < 1000)
-	{
-		g_pMain->GetServerResource(IDS_BAR4, &message, g_pMain->m_byExpEventAmount);
-		AppendNoticeEntry(pkt, elementCount, message.c_str(), "Juraid Mountain"); 
-	}
-
-			if (g_pMain->m_byExpEventAmount > 0)
```

Comment:
Can you please explain in the commit message why were these removed? This will be helpful to whoever revisit this in the future.